_articles/send/create-send.md

```diff
@@ -0,0 +1,227 @@
+2. To start creating a new Send, select the {% icon fa-plus %} **Create New Send** button:
+
+   {% image /send/send-create-webvault.png Send View in the Web Vault %}
+3. On the **Create New Send** dialog, specify the following to create the contents of your Send:
+
+   - **What type of Send is this?**: Specify whether this Send will be **Text** or a **File**:
+
+     |**Text**|Type or paste the desired text into the input box. Toggle the **When accessing the Send, hide the text by default** option to require recipients to [toggle visibility]({{site.baseurl}}/article/send-privacy/#hide-text) when they open a Send.|
+     |**File**|Select the **Choose File** button and browse for the file to send. The maximum file size per Send is 100 MB. (**Requires Premium**)|
+
+   - **Name**: Specify an identifiable, meaningful name for this Send.
+
+     By default, a Send will be scheduled for deletion 7 days from its creation. You may configure this and other options using the {% icon fa-chevron-down %} **Options** menu (see Step 4), otherwise select **Save** to finish creating your Send.
+
+4. Select the {% icon fa-chevron-down %} **Options** menu to configure the following options as desired:
+
+   |**Deletion Date**|The Send will be permanently [deleted]({{site.baseurl}}/article/send-lifespan/#deletion-behavior) on the specified date and time. By default, 7 days from creation. The **maximum allowed value** is 30 days from creation.|
+   |**Expiration Date**|The Send will [expire]({{site.baseurl}}/article/send-lifespan/#expiration-behavior) on the specified date and time. By default, **Never**.|
+   |**Maximum Access Count**|The Send will be [disabled]({{site.baseurl}}/article/send-lifespan/#maximum-access-count-behavior) after the specified access count is reached. By default, unspecified.|
+   |**Password**|[Require a password]({{site.baseurl}}/article/send-privacy/#passwords) to be entered by recipients of this Send in order to gain access.|
+   |**Notes**|Enter private notes for this Send, which will only be visible to the Sender.|
+   **Disable this send so that no one can access it**|Check this box to prevent this Send from being accessibly to any recipients. You will still be able to interact this Send from your Send view.|
+
+   Once you're happy with the configuration of your Send, select the **Save** button to finish.
+
+Once your Send is created, use the hover-over {% icon fa-cog %} gear dropdown and select the {% icon fa-copy %} **Copy Send link** button to copy the generated link to your clipboard:
+
+{% image /send/send-from-webvault.png Copy a Send link %}
+
+Once copied, share your Send link with intended recipients however you prefer. Sends are end-to-end encrypted, so you don't need to worry about exposing any Send data to whatever intermediary communications services you use.
```

Comment:
There should also be a check-box on the Create page that will automatically copy the link to the user's clipboard after Save.